index.js

```diff
@@ -5,14 +5,21 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "Blue"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+console.log('This is what the traveler object looks like: ', traveler)
+let img = document.querySelector(#profile.photo)
+let h2 = document.querySelector(#profile.name)
+let em = document.querySelector(#profile.nickname)
```

Comment:
Same thing here. Add the `name` and `nickname` classes to the appropriate elements inside the HTML file to make the suggestion below work.

```suggestion
let h2 = document.querySelector('#profile .name')
let em = document.querySelector('#profile .nickname')
```

If you want to try to search for the `h2` and the `em` without adding a class in the CSS, you could also do:

```suggestion
let h2 = document.querySelector('#profile h2')
let em = document.querySelector('#profile em')
```

So in the first line of code in the suggestion above -- I'm searching for an `h2` type of element that's nested inside another element with the id of `profile`. Similarly for the second line -- 'm searching for an `em` type of element that's nested inside another element with the id of `profile`.